### PR TITLE
More Qt6 builds

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 54edee0f735fe153516fad51924d49ef083893ff: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3490-a7c4b797fef7bf5520c43f71252891d8f18c75a5-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3490-a7c4b797fef7bf5520c43f71252891d8f18c75a5-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3490-a7c4b797fef7bf5520c43f71252891d8f18c75a5-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3490-a7c4b797fef7bf5520c43f71252891d8f18c75a5-x86-android.apk)
</details>

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -102,7 +111,7 @@ jobs:
           export LD_LIBRARY_PATH="${{ env.CMAKE_BUILD_DIR }}/vcpkg_installed/x64-linux/lib/":${LD_LIBRARY_PATH}
           cmake --build  "${{ env.CMAKE_BUILD_DIR }}" --target bundle --config ${{ env.BUILD_TYPE }}
           echo "ARTIFACT_PATHNAME=${{ env.CMAKE_BUILD_DIR }}/QField-x86_64.AppImage" >> $GITHUB_ENV
-          echo "ARTIFACT_NAME=qfield-${{ env.CI_PACKAGE_FILE_SUFFIX }}-linux-x64.AppImage" >> $GITHUB_ENV
+          echo "ARTIFACT_NAME=${{ env.CI_PACKAGE_FILE_BASENAME }}-linux}-${{ matrix.qt_version_suffix}}-x64.AppImage" >> $GITHUB_ENV
```
Comment (nirvn) [inline .github/workflows/linux.yml:114]: Are you sure about this?

Comment (m-kuhn) [reply .github/workflows/linux.yml:114]:
```suggestion
          echo "ARTIFACT_NAME=qfield-${{ env.CI_PACKAGE_FILE_SUFFIX }}-${{ matrix.qt_version_suffix}}-x64.AppImage" >> $GITHUB_ENV
```

better?

Comment (nirvn) [reply .github/workflows/linux.yml:114]: Yep that'll do. I will let you merge and keep my hands off this PR :)

**Comment by nirvn** [comment]
@m-kuhn , can we close this PR?

**Comment by m-kuhn** [comment]
It would be nice to re-run this against the current vcpkg version

**Comment by nirvn** [comment]
@m-kuhn , alright, converted to draft. Eventually rebasing against master would kill quite a few commits in here.

**Pull request closed without merging**